routes/index.js

```diff
@@ -1,12 +1,30 @@
 const express = require('express')
 const router = express.Router()
-const {Post} = require('../database')
-
+const {Post, Comment} = require('../database')
 
 router.get('/home', (request, response) => response.render('index'))
```

Comment:
home usually just lives at '/'. You don't usually go to google.com/home.
If you're meaning a dashboard or similar then I'd name it to reflect that.